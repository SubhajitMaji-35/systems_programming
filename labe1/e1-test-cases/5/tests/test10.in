programming
development
algorithm
function
variable
