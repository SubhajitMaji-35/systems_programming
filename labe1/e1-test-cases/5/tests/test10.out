rorammng
evlopmnt
lgrith
untion
arable
